doc/index.rst

```diff
@@ -39,6 +40,7 @@
 
     api/index.rst
     changes.md
+    Minimum Supported Versions <compatibility.rst>
```

Comment:
The name "Minimum Supported GMT/Python/NumPy Versions" are too long to fit the sidebar.